commands.py

```diff
@@ -66,7 +77,19 @@ def cmd_tell(self, c, msg, target, source):
             add_tell_message(source, location, recipient, message)
             reply("I'll save that message for %s" % recipient)
 
+    def cmd_check_tell(self, c, msg, target, source):
+    """ The first time a user interacts so the bot can see it, it will
+        check if there are new messages that the user has not yet been
+        notified of.
+    """
+        if get_told_messages(source) and not is_notified(source):
```

Comment:
Seems like it would be nice to make ``get_told_messages`` lazy, so operations like this one become cheaper.